browser/ui/BUILD.gn

```diff
@@ -215,8 +215,13 @@ source_set("ui") {
     }
 
     public_deps += [ "//brave/vendor/bat-native-ads" ]
-  }  # !is_android
-
+  } else {
+    # is_android
+    sources += [
+      "../../../chrome/browser/ui/webui/sanitized_image_source.cc",
+      "../../../chrome/browser/ui/webui/sanitized_image_source.h",
```

Comment:
I don't think that's a correct include, we need a lib deps for it, but I wonder why do we need that file at all?

Reply:
fixed (removed that)